Readme.md

```diff
@@ -1,10 +1,11 @@
 # Lombiq PowerShell Analyzers
 
+[![Lombiq.Analyzers.PowerShell NuGet](https://img.shields.io/nuget/v/Lombiq.Analyzers.PowerShell?label=Lombiq.Analyzers.PowerShell)](https://www.nuget.org/packages/Lombiq.Analyzers.PowerShell/)
```

Comment:
[Similar to .NET Analyzers](https://github.com/Lombiq/.NET-Analyzers/blob/dev/Docs/UsingAnalyzersDuringDevelopment.md#practices-on-suppressing-a-rule-for-a-given-piece-of-code), document how we use `SuppressMessageAttribute` here.